docs/core/porting/upgrade-assistant-wpf-framework.md

```diff
@@ -11,83 +11,434 @@ The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line to
+[09:51:01 INF] Adding Microsoft.Windows.Compatibility 6.0.0
+[09:51:01 INF] Recommending Windows TFM net6.0-windows because the project either has Windows-specific dependencies or builds to a WinExe
+[09:51:01 INF] Reference to .NET Upgrade Assistant analyzer package (Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers, version 0.3.261602) needs to be added
+[09:51:01 INF] Adding Microsoft.Windows.Compatibility 6.0.0
+[09:51:01 INF] Running analyzers on WebSiteRatings
+[09:51:02 INF] Identified 0 diagnostics in project WebSiteRatings
+[09:51:02 INF] Running analyzers on StarVoteControl
+[09:51:02 INF] Identified 0 diagnostics in project StarVoteControl
+```
 
-:::image type="content" source="media/upgrade-assistant-wpf-framework/configure-project.png" alt-text="Configure Windows Forms project in Visual Studio":::
+There's quite a bit of internal diagnostic information in the output, but some information is helpful. Notice that the analyze mode indicates that the upgrade will recommend that the project target the `net6.0-windows` target framework moniker ([TFM](../../standard/frameworks.md)). This is because the projects referenced by the solution are WPF projects, a Windows-only technology. A console application would probably get the recommendation to upgrade to TFM `net6.0` directly, unless it used some Windows-specific libraries.
 
-Review the created project and its files, especially its project file(s).
+If any errors or warnings are reported, take care of them before you start an upgrade.
 
-### Run upgrade-assistant
+## Run upgrade-assistant
 
-Open a terminal and navigate to the folder where the target project or solution is located. Run the `upgrade-assistant` command, passing in the name of the project you're targeting (you can run the command from anywhere, as long as the path to the project file is valid).
+Open a terminal and navigate to the folder where the target project or solution is located. Run the `upgrade-assistant upgrade` command, passing in the name of the project or solution you're upgrading.
+
+When a project is provided, the upgrade process starts on that project immediately. If a solution is provided, you'll select which project you normally run, known as the [upgrade entrypoint](#select-the-entrypoint). Based on that project, a dependency graph is created and a suggestion as to which order you should upgrade the projects is provided.
 
 ```console
-upgrade-assistant upgrade .\WpfTest.csproj
+upgrade-assistant upgrade .\WebSiteRatings.sln
+```
+
+The tool runs and shows you a list of the steps it will do. As each step is completed, the tool provides a set of commands allowing the user to apply or skip the next step or some other option such as:
+
+- Get more information about the step.
+- Change projects.
+- Adjust logging settings.
+- Stop the upgrade and quit.
+
+Pressing <kbd>Enter</kbd> will start the next step.
```

Comment:
To be clear, pressing enter will always choose the first item in the list of options. It's the same as typing '1' and then pressing enter.